pkg/util/env.go

```diff
@@ -59,7 +60,7 @@ func EnvInt(key string, def int) int {
 	if env := os.Getenv(key); env != "" {
 		val, err := strconv.ParseInt(env, 0, 0)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "WARNING: invalid env value (%v): using default, key=%s, val=%q, default=%d\n", err, key, env, def)
```

Comment:
Maybe klog instead?

Reply:
wasn't clear to me if nomos is using klog or not

Reply:
i think klog prints timestamps, and that isn't desired for the nomos CLI.

Reply:
This is used by container entrypoints as well. I think panicking may be acceptable for the CLI but seems like an undesirable behavior for backend components

Reply:
EntInt and EnvString are only called from package variable flag definitions, before the logger has been configured in `main()`. They can't use the logger, because some of the flags are used to configure the logger. AFAIK, there's no way to parse the flags in stages. Plus, if the service can't write to STDERR, it should probably crash anyway, because it was executed with a broken output stream.

Reply:
Ack